docs/data/tree-view/rich-tree-view/editing/editing.md

```diff
@@ -0,0 +1,22 @@
+---
+productId: x-tree-view
+title: Rich Tree View - Focus
+githubLabel: 'component: tree view'
+packageName: '@mui/x-tree-view'
+waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/treeview/
+packageName: '@mui/x-tree-view'
+---
+
+# Rich Tree View - label editing
+
+<p class="description">Learn how to edit the label of Tree View items.</p>
+
+:::warning
+this doc page is stip WIP
+:::
+
+To edit a label double click the item or press Enter on the keyboard
+
+Enter again after editing will save the new label. Bluring or pressing escape will cancel the editing
+
+{{"demo": "LabelEditing.js"}}
```

Comment:
A demo showing how to only edit leaves would be an interesting addition (you can check the drag & drop PR which has a very similar demo)